Code/skyrim_ui/src/app/components/settings/settings.component.html

```diff
@@ -5,11 +5,11 @@
       <h2>{{ 'COMPONENT.SETTINGS.AUDIO' | transloco }}</h2>
       <div class="flex-item">
         <label for="muted">{{ 'COMPONENT.SETTINGS.MUTE_UI' | transloco }}</label>
-        <app-checkbox
+        <input type="checkbox"
```

Comment:
put it in a new line for consistency
```suggestion
        <input
          type="checkbox"
```